### PR TITLE
Add Delete and Remove buttons

### PR DESCRIPTION
**Pull request by blase147**
In this project, I added  Delete and Add list button feature to the Todo list.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,39 +1,108 @@
...
+  localStorage.setItem('todos', JSON.stringify(todos));
+  displayTask();
+};
+addListForm.addEventListener('submit', addList);
 
+// remove task
+const removeList = (index) => {
+  const newlist = todos.filter((element) => element.index !== index);
+  todos.length = 0;
+  let i = 1;
```
Comment (BertrandConxy) [inline src/index.js:54]: - [ ] Good job 👏 👏 Your Add and Remove features are working. However, from the project requirements, they should be in a separate file. In other words, you are supposed to create a module containing add and remove functionality and then import it here in the `index.js` file for use 😃 👍  This is what looks more professional🏃 
This is the highlight among the project requirements 👇 
image

**Review by BertrandConxy — CHANGES_REQUESTED** [review]
Hi @blase147, 

Good job so far! 👏 👏 
There are some issues that you still need to work on to go to the next project but you are almost there! 🏃‍♂️ 🏃‍♂️ 


## Highlights 💪 
- No linter errors
- correct flow is used
- good PR

## Required Changes ♻️

- [ ] Good job so far 👏. I would suggest that you improve your README file by adding a good project description. Also, You may consider removing the sentences marked by (X) as in the picture 👇 
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @BertrandConxy in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -1,39 +1,108 @@
...
+  todos.push(...newlist);
+  localStorage.setItem('todos', JSON.stringify(todos));
+  displayTask();
+};
+todoList.addEventListener('click', (e) => {
+  if (e.target.classList.contains('btnDelete')) {
+    const index = parseInt(e.target.getAttribute('id'), 10);
+    removeList(index);
+  }
+});
```
Comment (RolandM99) [inline src/index.js:68]: - [ ] As requested by the previous reviewer, make sure that you create a new file as a module where you'll create these two functionalities for your app(Add and Remove). then you'll import that new file into your `index.js` file.

**Review by RolandM99 — CHANGES_REQUESTED** [review]
Hi @blase147 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**